packages/core-react/config/rollup.config.js

```diff
@@ -17,6 +19,18 @@ module.exports = {
     }
   },
   plugins: [
+    scss({
```

Comment:
I would strongly prefer not to add css processing to our build config. I find that things just get extremely confusing when your js bundler starts processing styles.

Reply:
this is mainly to get the scss exports to work. we should discuss that, since main motivation was to avoid duplicating css config in scss and JS